frontend/packages/llama-stack-modular-ui/frontend/src/app/Chatbot/hooks/useChatbotMessages.ts

```diff
@@ -106,7 +110,7 @@ const useChatbotMessages = ({
         setIsMessageSendButtonDisabled(false);
       }
     },
-    [isRawUploaded, modelId, selectedSourceSettings, systemInstruction],
+    [isRawUploaded, messages, modelId, selectedSourceSettings, systemInstruction],
```

Comment:
Might not related to the PR but since you made the change here - I don't see any benefits of using the `React.useCallback` on this function, maybe we could get rid of it.
It's called [here](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/packages/llama-stack-modular-ui/frontend/src/app/Chatbot/ChatbotMain.tsx#L135-L139), which only triggers when you click on the send button, thus, I don't see a value to memoize the callback.